scripts/flattener.py

```diff
@@ -19,6 +19,9 @@
 import json
 import numbers
 import flattener_mods as fm
+import PIL
```

Comment:
Is this needed? Seems like only `Image` is used, and that is imported below
(would need to update to `Image.MAX_IMAGE_PIXELS = 933120000`